### PR TITLE
add layer_norm (paddle vs torch) scripts

### PR DESCRIPTION
**Pull request by windstamp**
add layer_norm (paddle vs torch) scripts

### CODE REVIEW

**Comment by windstamp** [comment]
pytorch layer_norm

image

**Comment by windstamp** [comment]
paddle layer_norm

File: api/tests_v2/configs/layer_norm.json [hunk 1]
```diff
@@ -11,4 +11,18 @@
             "type": "Variable"
         }
     }
+}, {
```
Comment (Xreki) [inline api/tests_v2/configs/layer_norm.json:14]: 为什么要加一个一样的配置？

Comment (windstamp) [reply api/tests_v2/configs/layer_norm.json]: thanks, done.

**Review by Xreki — APPROVED** [review]

**Pull request merged**